frontend/packages/metal3-plugin/src/plugin.ts

```diff
@@ -138,6 +141,23 @@ const plugin: Plugin<ConsumedExtensions> = [
       required: [FLAGS.BAREMETAL, METAL3_FLAG],
     },
   },
+  {
+    type: 'Dashboards/Overview/Activity/Resource',
+    properties: {
+      k8sResource: {
+        isList: true,
+        kind: referenceForModel(NodeMaintenanceModel),
+        prop: 'maintenances',
+      },
+      isActivity: (resource) => _.get(resource.status, 'phase') !== 'Succeeded',
```

Comment:
IIUC the maintenance phase can be `Running|Success|Error` so I'd go for 
```suggestion
      isActivity: (resource) => _.get(resource.status, 'phase') === 'Running',
```
just to be sure.